### DISCUSSION TITLE
Firewall Ports

### DISCUSSION THREAD

**Discussion by knylbyte** [arendst/Tasmota#11330] [Q&A] [answered]
Hey everyone,

i need some help for my firewall config. I separated my iot network from my mainnetwork and block every traffic to the other networks but want to access the webinterface from my main network. So i opened Port 80 TCP/UDP from my iot network to the main network, but can't access the tasmota webinterface from the main network. There are no restriction for the main network to any other network. I also opened port 1883 TCP for the mqtt communication and it works like a charm. Maybe someone can help me for the webinterface.

**Comment by barbudor** [comment, marked as answer]
Browsing the tasmota GUI from your PC is a connection going from main network to IOT network.
I understand that you opened the wrong direction, allowing the devices to access web servers located on your main network.

Firewall is not all. You may also miss a route.
If the router defined as your default gateway on your PC doesn't know that request to your IOT network instead of the Internet, it won't work.

Without knowing more details on your setup, can only provide very general tips.

**Reply by knylbyte** [reply]
I'am new to Ubiquiti Hardware and want to understand the firewall rules. 
Every network has the USG as the default gateway with different ip-areas. 
For example:
main network: 10.0.10.1/24
iot network: 10.0.30.1/24
firewall -> LAN IN: drop all from IoT network to Group (192.168.0.0/24 & 172.16.0.0/12 & 10.0.0.0/8) 
If I enable this rule I cant acces to the IoT network from my main network.
If I disable this rule I can acces to the IoT network. 

I hope this helps?

**Reply by barbudor** [reply]
So the last sentence seems to mean you have solved it ?
For support on Ubiquity, you'd better go to a dedicated forum where you will find more experienced users.
Not really a Tasmota problem.

**Reply by knylbyte** [reply]
Yes meanwhile I solved my problem but thanks for your help. It was my firewall rule
